app/api-client.js

```diff
@@ -30,6 +30,9 @@ module.exports = (function () {
 
 
   function endpoint_url(endpoint) {
+    if (endpoint && !endpoint.endsWith('/')) {
```

Comment:
Some thoughts on this (don't have to change anything if you don't want)

We should probably always pass the endpoint in to `api_request()`, I wonder if here default the `endpoint` param to `/` would be defensive enough.

If you have a default and this check you may also get rid of the `(endpoint || '')` below, and maybe just use string interpolation, e.g.

```javascript
return url.resolve(`${config.api.base_url}/${endpoint}`)
```

BTW, I like that we check if ends with `/` better :)

Reply:
Yep, I haven't been writing ES6, but I will.

Reply:
I guess I'm going to have to bite the bullet and start ES6ing at some point...

Reply:
@xoen in this case, your suggestion doesn't work, because `url.resolve` does the concatenation of the URL parts - there are two parameters, not one.

Reply:
Mainly was a point around testing if the `endpoint` was present and use of ES6 concatenation, etc...but you got the idea 🖖